src/lib/validators/probability.ts

```
@@ -1,17 +1,17 @@
-      'Likelihood that the message is relevant (greater than 0.5 means related, less than 0.5 means not related)',
-    ),
-  reason: z
-    .string()
-    .min(1)
-    .describe(
-      'Explanation for why the message is considered relevant / not relevant',
-    ),
-});
-
-export type Probability = z.infer<typeof probabilitySchema>;
+import { z } from 'zod/v4';
```

ESLint

[error] Unable to resolve path to module 'zod/v4'.

(import-x/no-unresolved)